clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -111,18 +99,32 @@
       In: ${arg}.cu
       Out: ${arg}.dp.cpp
 
-- Rule: rule_user_defined_function
+- Rule: rule_user_defined_function_cpp
   Kind: CMakeRule
   Priority: Fallback
   MatchMode: Partial
-  CmakeSyntax: cuda_files_in_user_defined_function
+  CmakeSyntax: cuda_files_in_user_defined_function_cpp
+  In: ${func_name}(${value})
+  Out: ${func_name}(${value})
+  Subrules:
+    value:
+      MatchMode: Full
+      In: ${arg}.cpp
```

Comment:
```suggestion
      In: ${arg}.cpp | ${arg}.cxx | ${arg}.cu | ${arg}.C
```

Reply:
Will implement this feature in a separate PR.